acceptance-tests/src/main/java/it/com/atlassian/bitbucket/jenkins/internal/util/BitbucketUtils.java

```diff
@@ -40,6 +45,16 @@ public class BitbucketUtils {
     public static final String REPO_ADMIN_PERMISSION = "REPO_ADMIN";
     public static final String REPO_SLUG = "rep_1";
 
+    public static final String PRIV_KEY = "-----BEGIN OPENSSH PRIVATE KEY-----\n" +
+                                          "b3BlbnNzaC1rZXktdjEAAAAABG5vbmUAAAAEbm9uZQAAAAAAAAABAAAAMwAAAAtzc2gtZW\n" +
+                                          "QyNTUxOQAAACAL2dUHG4kSVqZ9pOqBhtEjygdxuiYPbVeyUMX6IBPBJQAAAJAOD47NDg+O\n" +
+                                          "zQAAAAtzc2gtZWQyNTUxOQAAACAL2dUHG4kSVqZ9pOqBhtEjygdxuiYPbVeyUMX6IBPBJQ\n" +
+                                          "AAAEA4u5H/tXLBU8F5SqbNZi+KRZZ+GPFjNio8HKLXm4p+tQvZ1QcbiRJWpn2k6oGG0SPK\n" +
+                                          "B3G6Jg9tV7JQxfogE8ElAAAADHRlc3RAamVua2lucwE=\n" +
+                                          "-----END OPENSSH PRIVATE KEY-----\n";
+    public static final String PUB_KEY =
+            "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIAvZ1QcbiRJWpn2k6oGG0SPKB3G6Jg9tV7JQxfogE8El test@jenkins\n";
```

Comment:
```suggestion
            "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIAvZ1QcbiRJWpn2k6oGG0SPKB3G6Jg9tV7JQxfogE8El\n";
```

I don't think we need a label on it?